docs/reference/model.rst

```diff
@@ -1,7 +1,53 @@
-.. automodule:: stravalib.model
-   :members:
-   :undoc-members:
-   :show-inheritance:
+.. automodule::
```

Comment:
`stravalib/docs/reference/model.rst:53:<autosummary>:1: ERROR: Content block expected for the "raw" directive; none found.` 

getting this error now. normally that error measn that i'm missing a space after a raw directive 

```
.. automodule::
    stravalib.model
```

vs

```
.. automodule::

    stravalib.model
```

but when i do it the right way, the page doesn't render. so i'm trying to find a balance between rendering the module titles correctly but also rendering the doc page correctly.

Reply:
`make -C docs clean` and re-serving seems to resolve this problem at my end.